examples/compareLambdaTypes.py

```diff
@@ -25,30 +27,31 @@
 
 CFHTTORUN = 'cal-53535-i-797722_1'
 
+
 class DiffimTestCases(unittest.TestCase):
```

Comment:
Hmm, it runs (once I add `.fits` to the hardwired afw filenames), but it encounters errors: (1) it calls `ipDiffim.ImageStatisticsF()` without a required `policy` positional argument - this may be related to DM-7299? (2) it fails testLeaks with 1798 blocks of memory leaks - not sure if this matters or not?